src/app/social-media-challenge/social-media-challenge.component.html

```diff
@@ -20,23 +20,22 @@ <h4>Rules to participate</h4>
 
                 <ul>
                     <li>
-                        Take a picture of people, or your favorite pet if you want :) <em>showing the NG-BE logo</em>
+                        Take a funny or happy picture of yourself, someone else or even your favorite pet, <em>showcasing the NG-BE logo</em>.
                     </li>
                     <li>
-                        Post it on social media ( Facebook, Twitter or LinkedIn ) <em>before 1/08/2018</em>
+                        Post your picture on social media ( Facebook, Twitter or LinkedIn ) <em>before August 1, 2018.</em>
                     </li>
                     <li>
-                        <em>Tag NG-BE in the post</em> so that we are aware of your submission.
-                        Make sure your post is public so that we can review it.
+                        <em>Use #ngbe2018 or @ngbeconf in your post</em> so that we are alerted of your submission. Make sure your post is public so that we can review it.
                     </li>
                     <li>
-                        The 3 most original pictures <em>win a 25% discount on a workshop or conference ticket</em>
+                        The 3 most original pictures <em>win a 25% discount on a workshop or conference ticket.</em>
                     </li>
                     <li>
-                        Submissions are <em>selected objectively</em> by the members of the NG-BE core team
+                        Submissions are <em>rated democratically</em> by the members of the NG-BE core team.
                     </li>
                     <li>
-                        The 3 winners are announced on the 10th of Augustus 2018.
+                        The 3 winners are announced on the August 10, 2018.
```

Comment:
@jvandemo `on August 10`?

Reply:
@samvloeberghs You are right, it should be "on August 10, 2018", nice catch 👍